test/Bootstrapper.FunctionalTests/BootstrapperTests.cs

```diff
@@ -118,7 +117,7 @@ public void BootstrapperInvokesApplicationHostWithExplicitAppBase(string flavor,
         {
             var runtimeHomeDir = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
 
-            using (var tempSamplesDir = TestUtils.PrepareTemporarySamplesFolder(runtimeHomeDir))
+            using (var tempSamplesDir = TestUtils.PrepareTemporarySamplesFolder(runtimeHomeDir, globalJson: false))
```

Comment:
The tradition `ProjectResolver` not only look at one project but also it's sibling folder therefore `HelloShare` is resolved. The problem with adding a global.json here is that it's adding unnecessary paths to the `DNX` source code path (https://github.com/aspnet/dnx/blob/victorhu/require-servicing/test/Microsoft.Dnx.CommonTestUtils/TestUtils.cs#L246). As a result when restore the sample application dependences like `Microsoft.Dnx.Compilation.Abstraction` will be resolved as project instead of package.